docs/build/quick-start/hello-world.md

```diff
@@ -63,7 +69,7 @@ It will give you options for different types of projects but for this tutorial c
 
 ::: info
 
-The private key of your wallet will be included in the `.env` file of the project and won't be pushed to GitHub.
+The private key of your wallet will be included in the `.env` file of the project and won't be pushed to GitHub. However, as this means the private key is in plain text, it's recommended to use a private key that isn't associated with any real funds.
```

Comment:
I'm fine with this single change for now, but we should update all tutorials that use this same `::: info` tag with this additional warning.

Reply:
Issue created: https://github.com/matter-labs/zksync-web-era-docs/issues/985